miner/builder_test.go

```diff
@@ -269,7 +271,7 @@ func TestBlockBuilder_CreateBlockFlow(t *testing.T) {
 		assert.True(t, ContainsAtx(b.ATXIDs, atxs[2].ID()))*/
 
 		assert.Equal(t, []types.ATXID{atx1, atx2, atx3, atx4, atx5}, *b.ActiveSet)
-	case <-time.After(1 * time.Second):
+	case <-time.After(1 * time.Minute):
```

Comment:
The other fix makes a lot of sense to me. I'm also ok with increasing the timeout, but I'm wondering if it isn't just a leftover from a first attempt at a fix - does it really take more than a second sometimes?

Reply:
@noamnelke Thanks for noticing. No, it's not a leftover. Even with the other fix, sometimes, under high CPU load, spawning a goroutine takes multiple seconds, so the timeout occurs. I've run the test around 5M times and it took around 15 seconds around 2 times per 1M iterations. So, I think, 1 minute would give some reserve.